src/libs/ReportUtils.ts

```diff
@@ -4248,6 +4257,7 @@ function buildOptimisticTaskCommentReportAction(
 
     // These parameters are not saved on the reportAction, but are used to display the task in the UI
     // Added when we fetch the reportActions on a report
+    // eslint-disable-next-line
```

Comment:
can we put the name of the error we're disabling here?

Reply:
Discussing [here](https://github.com/Expensify/App/pull/49325#discussion_r1767627021)